### PR TITLE
topology: Add Waves codec to mt8186 topology

### PR DESCRIPTION
**Pull request by barry-waves**
config: Add Waves override config for mt8186_chrome.
module_adapter: Enable module_adapter and Waves codec.

Signed-off-by: barry.jan <barry.jan@waves.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

**Comment by lgirdwood** [comment]
test this please

**Comment by lgirdwood** [comment]
SOFCI TEST

**Review by lgirdwood — APPROVED** [review]
@ranj063 any comments ?

**Comment by lgirdwood** [comment]
CI did not fully run. Lets try again.

**Comment by lgirdwood** [comment]
test this please

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
CI has been down - rerun

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@thesofproject/mediatek fyi - this is for you guys. CI is fine, I just noticed it's for your branch. Pls review/merge.

**Comment by lgirdwood** [comment]
adding @thesofproject/google too

**Review by kuanhsuncheng — APPROVED** [review]

File: src/arch/xtensa/configs/override/mt8186_chrome_waves.config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CONFIG_CAVS_LPS=y
```
Comment (yaochunhung) [inline src/arch/xtensa/configs/override/mt8186_chrome_waves.config:1]: Is it required for mt8186?

**Comment by barry-waves** [comment]
I have sent another pull request in main branch instead.

**Pull request closed without merging**